app/controllers/api/reporting/totals_controller.rb

```diff
@@ -62,4 +71,53 @@ def set_default_filters
   def set_filters
     @filters = ReportingAPI::EventFilter.new(params:, filters: FILTERS)
   end
+
+  def set_scope
+    organisation_id = current_user.organisations
```

Comment:
I don't think this looks right: `organisations` will get all the organisations associated to a user, but then we save it to a variable called `organisation_id` which suggests we want a single integer ID value.